linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,rhv4,sles15,wrlinux1019
```

Comment:
s/sles/sle/g

Reply:
Forgot to push after I fixed it last night.  I apologize